libs/hwManager/manufacturers/trezor.js

```diff
@@ -1,39 +1,73 @@
 /* istanbul ignore file */
 /* eslint-disable no-bitwise */
+// TODO remove this line once we add constants file for this module
 import { models } from '../../../src/constants/hwConstants';
 
 const listener = (transport, actions) => {
+  // TODO use contants instead of hardcoded text
   transport.on('connect', (device) => {
     actions.add({
       deviceId: device.features.device_id,
       label: device.features.label,
       model: device.features.model === '1' ? models.trezorOne : models.trezorModelT,
       path: device.originalDescriptor.path,
-      manufactor: 'trezor',
+      manufactor: 'trezor', // TODO use contants instead of hardcoded text
     });
   });
 
+  // TODO use contants instead of hardcoded text
   transport.on('error', (error) => {
     // eslint-disable-next-line no-console
     console.error('Trezor Error: ', error);
   });
 
+  // TODO use contants instead of hardcoded text
   transport.on('disconnect', (device) => {
     actions.remove(device.originalDescriptor.path);
   });
 
+  // TODO use contants instead of hardcoded text
   process.on('exit', () => {
     transport.onbeforeunload();
   });
 };
 
+// TODO move this to utils file
 const hardeningConstant = 0x80000000;
 const getHardenedPath = index => [
   (44 | hardeningConstant) >>> 0,
   (134 | hardeningConstant) >>> 0,
   (index | hardeningConstant) >>> 0,
 ];
 
+// eslint-disable-next-line max-statements
```

Comment:
I think now is a good time to simplify this function and remove `eslint-disable` from here. I'd consider one with object destructuring, something like:
```js
const toTrezorGrammar = ({senderPublicKey, requesterPublicKey, recipientId, ...rest }) => {
const tx = {
  ...rest,
  recipient_id: recipientId,
  sender_public_key: senderPublicKey,
  requester_public_key: requesterPublicKey,
  ...
}
Object.keys(tx).forEach((key) => (!tx[key]) && delete tx[key]);

return tx;
```

Reply:
Agree, I made a few changes to work properly but yes this was need it.